charts/apps/immich/values.yaml

```diff
@@ -99,19 +99,20 @@ machine_learning:
           port: 3003
           protocol: HTTP
 
-  command: "/bin/sh"
+  command: "python"
   args:
-    - "./entrypoint.sh"
+    - "src/main.py"
```

Comment:
I think we can skip defining this entirely since it's set in the Dockerfile

Reply:
let me test that

Reply:
LGTM